hugolib/author.go

```diff
@@ -13,23 +13,51 @@
 
 package hugolib
 
+import (
+	"fmt"
+	"regexp"
+	"sort"
+	"strings"
+
+	"github.com/spf13/cast"
+)
+
+var (
+	onlyNumbersRegExp = regexp.MustCompile("^[0-9]*$")
+)
+
 // AuthorList is a list of all authors and their metadata.
 type AuthorList map[string]Author
 
+// Authors returns a slice of authors sorted by ID
+func (al AuthorList) Authors() []Author {
```

Comment:
The default sort on `Authors`, i.e. only one sort, can be implemented as methods on `Authors`. The additional sorts, which is now out of scope, could be implemented similar to the `pageSorter`. As to the `Desc` funcs, it is cleaner in my eye to just chain the methods, like `ByName().Reverse()`. Also, I believe we use `sort.Stable` all (?) other places.